.github/workflows/ci.yml

```diff
@@ -32,6 +32,26 @@ jobs:
         name: ${{ matrix.host }}-x
         path: bin/x${{ matrix.host == 'windows-latest' && '.exe' || '' }}
 
+  # Github doesn't have runners for anything but 64 bit
+  build-32bit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: hecrj/setup-rust-action@v1
+      with:
+        targets: "i686-unknown-linux-gnu"
+    - name: multilib
+      run: sudo dpkg --add-architecture i386 && sudo apt-get update && sudo apt-get install gcc-multilib libssl-dev:i386
+    - run: cargo install --path xbuild --root . --target i686-unknown-linux-gnu
+      env:
+        OPENSSL_LIB_DIR: /usr/lib/i386-linux-gnu/
+        OPENSSL_INCLUDE_DIR: /usr/include/openssl/
+
+    - uses: actions/upload-artifact@v3
+      with:
+        name: i686-x
+        path: bin/x
```

Comment:
Might be nice to add the `.exe` suffix for Windows... And add a 32-bit Windows build?